Source/Csla.Channels.RabbitMq/RabbitMqProxy.cs

```diff
@@ -33,9 +33,15 @@ public RabbitMqProxy(ApplicationContext applicationContext, RabbitMqProxyOptions
         throw new ArgumentNullException(nameof(options));
 
       DataPortalUrl = options.DataPortalUrl;
-      Timeout = (int)options.Timeout.TotalSeconds;
+      _timeout = options.Timeout;
     }
 
+    private readonly TimeSpan _timeout;
+    /// <summary>
+    /// Gets or sets the timeout value in seconds for the RabbitMQ request.
```

Comment:
The value is not necessarily in seconds. It's a time span and how that's used by rabbit mq should here be irrelevant.